Enable opt-out of browserSync (#31)
spothero/ace: docs/settings-browsersync.md

---

**key:** `browserSync`

> Optional.

Reviewer: dont need to define optional, not a convention used in other spots.

> When included, browserSync can watch for changes in files not assocated with webpack (such as stylesheets compiled by a gulp task) and hot-reload the browser.

Reviewer: ```suggestion
When included, BrowserSync can watch for changes in files not associated with webpack (such as stylesheets compiled by a Gulp task) and inject them into browser.
```
